packages/rollup-dev-tools/src/tools/transpiler/transpiler.ts

```diff
@@ -658,12 +659,20 @@ export class TranspilerImpl implements Transpiler {
           Opcode.getCodeNumber(opcodeAndBytes.opcode) + 1
         )
       }
+      let transpiledBytecodeReplacement: EVMBytecode
+      if (
+        insideUnreachableCode 
+        || !this.opcodeReplacer.shouldReplaceOpcode(opcodeAndBytes.opcode)
+      ) {
+        transpiledBytecodeReplacement = [opcodeAndBytes]
+      } else {
+        // record that we will need to add this opcode to the replacement table
+        replacedOpcodes.add(opcodeAndBytes.opcode)
```

Comment:
Should this accept `EVMBytecode`? Could be premature optimization, but I could see a world in which we replace repeated bytecode with these "function" calls as well when the result would be less deployed bytecode.

Reply:
I'm gonna leave for now as the functionality this PR improves was already only able to act on individual opcodes.  Have to think more about that though, interesting thought!